spec/integration/util/windows/user_spec.rb

```diff
@@ -79,6 +79,7 @@
     let(:username) { 'fabio' }
     let(:bad_password) { 'goldilocks' }
     let(:logon_fail_msg) { /Failed to logon user "fabio":  Logon failure: unknown user name or bad password./ }
+    let(:password){'test'}
```

Comment:
This let is defined here but is never used in this scope.

Reply:
Good catch, I removed the unused password.